Examples/Jax/jax_sr_example.py

```diff
@@ -1,38 +1,58 @@
 import netket as nk
 import numpy as np
 import jax
-from jax.experimental.optimizers import sgd as JaxSgd
 import cProfile
 
 # 1D Lattice
 L = 20
 g = nk.graph.Hypercube(length=L, n_dim=1, pbc=True)
 
+
 # Hilbert space of spins on the graph
 hi = nk.hilbert.Spin(s=0.5, graph=g)
 
 ha = nk.operator.Ising(h=1.0, hilbert=hi)
 
 alpha = 1
 ma = nk.machine.JaxRbm(hi, alpha, dtype=complex)
+ma2 = nk.machine.JaxRbm(hi, alpha, dtype=complex)
```

Comment:
I don't get the point of this example.
Are you trying to show that you get exactly the same result?

Otherwise I think it's better to create another example file.

Reply:
Yes, I abused this example a bit for testing, I think once we figured out the interface we can restore the old example and add a new one for the on-the-fly version

Reply:
I removed the commit when I rebased it, so the example is again the way it was before.